input.go

```diff
@@ -18,6 +18,13 @@ import (
 	"github.com/hajimehoshi/ebiten/internal/ui"
 )
 
+// InputChars returns a slice of Unicode "printable" runes read from the keyboard.
+//
+// This function is concurrent-safe.
+func InputChars() []rune {
+	return ui.CurrentInput().RuneBuffer()
```

Comment:
I think the result should be copied or the caller of `InputChars` can modify the internal buffer.